paparazzi/extensions/accessibility/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.kotlin.kapt'
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+
+dependencies {
+  api project(':paparazzi')
```

Comment:
what types does this need from paparazzi?  might be worth pulling into a commons?